exonum-java-binding-core/src/test/java/com/exonum/binding/proxy/NativeHandleTest.java

```diff
@@ -64,4 +64,12 @@ void closeMultipleTimes() {
     nativeHandle.close();
     assertFalse(nativeHandle.isValid());
   }
+
+  @Test
+  public void toStringHexRepresentation() {
+    long handle = 0x11L;
+    nativeHandle = new NativeHandle(handle);
+
+    assertThat(nativeHandle.toString()).contains(Long.toHexString(handle).toUpperCase());
```

Comment:
Please do not use such complex expressions as expected values `Long.toHexString(handle).toUpperCase()` instead of "11" or "0x11" — I have no idea what is the expected format from this expression.